test/Microsoft.Azure.Relay.UnitTests/DisplayTestNameAttribute.cs

```diff
@@ -4,14 +4,13 @@
 namespace Microsoft.Azure.Relay.UnitTests
 {
     using System.Reflection;
-    using Microsoft.Extensions.PlatformAbstractions;
     using Xunit.Sdk;
 
     public class DisplayTestMethodNameAttribute : BeforeAfterTestAttribute
     {
         public override void Before(MethodInfo methodUnderTest)
         {
-            TestUtility.Log($"Begin {methodUnderTest.DeclaringType}.{methodUnderTest.Name} on {PlatformServices.Default.Application.RuntimeFramework.ToString()}");
```

Comment:
>PlatformServices.Default.Application.RuntimeFramework. [](start = 95, length = 54)

This was returning values with incorrect version numbers in them. Presumably since it's from 2016.